### DISCUSSION TITLE
Not Getting Response in Chat UI with Local Ollama Instance

### DISCUSSION THREAD

**Discussion by jim380** [ragapp/ragapp#218] [Q&A]
**Describe the bug**
Not getting any response back from a local Ollama instance that is running fine and gets correctly picked by Ragapp.

**To Reproduce**

1. Deploy with `MODEL=llama3 docker compose up -d`
2. Go to `:8000`, type something in the text box and hit send. Nothing is returned
3. Go to `:8000/admin`. type something in the text box and hit send. Same thing

**Expected behavior**
The user should get a response back in chat.

**Screenshots**

https://github.com/user-attachments/assets/a6f1eaed-d594-41e4-aecc-94ae25f9f41d

**Desktop (please complete the following information):**
 - OS: Ubuntu 22.04
 - Browser: Chrome 128.0.6613.138

**Additional context**
[ollama_logs.txt](https://github.com/user-attachments/files/17109814/ollama_logs.txt)

**Comment by marcusschiesser** [comment]
logging at the log, the issue is that ollama with llama3:latest doesn't support function calling:
```
   |   File "/usr/local/lib/python3.11/site-packages/ollama/_client.py", line 488, in _request
    |     raise ResponseError(e.response.text, e.response.status_code) from None
    | ollama._types.ResponseError: llama3:latest does not support tools
```

**Reply by jim380** [reply]
> logging at the log, the issue is that ollama with llama3:latest doesn't support function calling:
> 
> ```
>    |   File "/usr/local/lib/python3.11/site-packages/ollama/_client.py", line 488, in _request
>     |     raise ResponseError(e.response.text, e.response.status_code) from None
>     | ollama._types.ResponseError: llama3:latest does not support tools
> ```

Understood. The issue here is though, that a fresh deployment with `MODEL=llama3 docker compose up -d` doesn't even work, with no config changed via the admin UI. As demonstrated in the recording, it's just a "hello" in the chat UI.
